### PR TITLE
User story 34/shape drawing tool

### LINKED ISSUE #34: As a player, I can add shapes, so that I can draw specific shapes better

### PR DESCRIPTION
**Pull request by ChristopherPearse**
Adds functionality that includes a shapes option in the tool bar that allows you to draw various shapes.
Shapes include:
- Square
- Circle
- Triangle
- Star

Closes #34

### CODE REVIEW

File: src/public/css/drawStyle.css
Comment (RudolphTorlage) [inline src/public/css/drawStyle.css:1]: Consistent sizing of the buttons

File: src/public/html/drawingBoard.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h1>How to play</h1>
           width: 100%;
           height: 100%;
           background-color: rgba(0, 0, 0, 0.5);
-          z-index: 0;
+          z-index: 1000;
```
Comment (RudolphTorlage) [inline src/public/html/drawingBoard.html:75]: This makes buttons underneath unclickable. 1000 seems big, was 1 already in use?

File: src/public/html/drawingBoard.html [hunk 2]
```diff
@@ -242,18 +269,16 @@ <h1>Voting</h1>
...
-    </script>
-    <script src=”https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js”
-    integrity=”sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q”
-    crossorigin=”anonymous”>
-    </script>
-    <script src=”https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js”
-    integrity=”sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl”
-    crossorigin=”anonymous”>
-    </script>
+    <button onclick="window.location.href='/logout';" style="position: fixed; bottom: 10px; right: 10px; z-index: 100000;">Logout</button>
```
Comment (RudolphTorlage) [inline src/public/html/drawingBoard.html:272]: Implemented bootstrap? What was the previous code doing?

File: src/public/js/drawingBoard.js [hunk 3]
```diff
@@ -395,6 +399,38 @@ sprayPaintButton.addEventListener('click', () => {
   drawingTool = 'sprayPaint'
 })
 
+squareButton.addEventListener('click', () => {
+  if (drawColour == 'white') {
+    changeColour('black')
```
Comment (RudolphTorlage) [inline src/public/js/drawingBoard.js:404]: Does this reset the color back to black every time a new shape is chosen? Is this what we want?

File: src/server.js [hunk 4]
```diff
@@ -83,49 +83,76 @@ io.on('connection', (socket) => {
...
-    room.todo.push(data)
-    if (room.members.length !== room.gameSize) {
-      // wait for everyone to join, then do this process
-      return
-    }
-
-    sendPrompts(room)
-  })
-
-  function sendPrompts(room) {
```
Comment (RudolphTorlage) [inline src/server.js:105]: This function has been deleted and not replaced with something, is this intended?

File: src/server.js [hunk 5]
```diff
@@ -83,49 +83,76 @@ io.on('connection', (socket) => {
...
-
-      rooms[roomId].prompts[socketID] = prompt
-      updateGridSubmission(
-        roomId,
-        users.get(socketID).username,
-        'prompt',
-        prompt,
-        socketID
-      )
+  // socket.on('inputDone', (data) => {
```
Comment (RudolphTorlage) [inline src/server.js:86]: Why has all of the commented code been left in the codebase?

File: src/public/html/drawingBoard.html (hunk 1 above)
Comment (ChristopherPearse) [reply src/public/html/drawingBoard.html:75]: Yes, values smaller than 1000 are already in use, 1000 also allows for future components to be made between the two layers.

**Review by RudolphTorlage — COMMENTED** [review]
Please address the comments in the review before I can merge, thank you.

File: src/public/html/drawingBoard.html (hunk 1 above)
Comment (RudolphTorlage) [reply src/public/html/drawingBoard.html:75]: OK, perhaps this should be standardized in a future update, but leaving room for other features has no harm in it.

File: src/public/html/drawingBoard.html (hunk 2 above)
Comment (ChristopherPearse) [reply src/public/html/drawingBoard.html:272]: The previous code was not actually incorporating bootstrap. As can be seen the quotation marks around the new implementation are different; this is the correct way of incorporating it.

Comment (RudolphTorlage) [reply src/public/html/drawingBoard.html:272]: Thank you for fixing that

File: src/server.js (hunk 5 above)
Comment (ChristopherPearse) [reply src/server.js:86]: This was done as i was not 100% sure if the new implementation would work, thus I left it in incase we have to revert back to it.

File: src/server.js (hunk 4 above)
Comment (ChristopherPearse) [reply src/server.js:105]: The functionality of this function has been moved to be done within the inputDone receipt from the sockets. This was done to remove errors that the function was creating.

Comment (RudolphTorlage) [reply src/server.js:105]: OK, as long as the functionality has been accounted for

**Pull request merged**